packages/components/tabs/src/Tab.tsx

```diff
@@ -64,26 +62,23 @@ function _Tab(
     onKeyPress: handleKeyPress,
     style,
     testId,
-    tabIndex,
+    tabIndex: isSelected ? 0 : -1,
   };
 
-  if (isDisabled) {
-    elementProps['aria-disabled'] = true;
-  }
-
   elementProps['aria-selected'] = isSelected;
   elementProps['role'] = 'tab';
   elementProps['aria-controls'] = panelId;
   return (
-    <Box
-      as="button"
+    <Button
```

Comment:
I think it would be easier to use a default html button here instead our custom one, so not to reset all the styles., like:
![image](https://user-images.githubusercontent.com/10744462/133107569-887bec53-59de-4609-8bb2-d023e3643e35.png)

Reply:
What if we use the `transparent` variant?